### PR TITLE
Feature/registration form

### PR DESCRIPTION
**Pull request by Jacu93**
Please check the current status of my work as I wasn't able to finish backend input validation today.

I would like to inform the user about successful registration by displaying short message under the form. I would like to have this message in green. Normally it is used to present validation errors so in styles.css it's defined as red.
However when I try to add this short message by using innerHTML and change color by using style.color it gives an error saying that function register () is undefined. This part is commented in the latest commit.

### CODE REVIEW

File: src/main/resources/static/scripts/register.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      method: 'POST',
+      body: JSON.stringify(data),
+      headers:{
+        'Content-Type': 'application/json'
+      }
+    }).then(res => res.json())
+    .then(response => console.log('Success:', JSON.stringify(response)))
+    .catch(error => console.error('Error:', error));
+
+    //ten blok ponizej mi nie dziala :( wywala cala funkcje register()
```
Comment (piekaa) [inline src/main/resources/static/scripts/register.js:34]: pls don't use PL even in comments ;p

File: src/main/resources/static/scripts/register.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    /*document.getElementById("error").innerHTML = "Account registered successfully!";
+    document.getElementById("error").style.color = #009933;*/
```
Comment (piekaa) [inline src/main/resources/static/scripts/register.js:36]: style.color = "#009933"; instead of style.color = #009933

Comment (piekaa) [inline src/main/resources/static/scripts/register.js:36]: it should be executed in response callback lambda function

File: src/main/resources/static/scripts/register.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    var emailElement=document.getElementById("emailInput");
+    var nameElement=document.getElementById("nameInput");
+    var passwordElement=document.getElementById("passInput");
+    var repasswordElement=document.getElementById("repassInput");
+
+    var url = 'http://localhost:8080/auth/signup';
+    var data = {"email": emailElement.value, "name": nameElement.value, "password": passwordElement.value};
+
+    if (passwordElement.value != repasswordElement.value){
```
Comment (piekaa) [inline src/main/resources/static/scripts/register.js:11]: wherever you can, you should compare using === or !== in JavaScript
https://dorey.github.io/JavaScript-Equality-Table/

**Review by piekaa — APPROVED** [review]

File: src/main/resources/static/scripts/register.js (hunk 2 above)
Comment (Jacu93) [reply src/main/resources/static/scripts/register.js:36]: `.then(response => console.log('Success:', JSON.stringify(response)))`

It should be somewhere around here?

**Pull request merged**